plugin/build.gradle.kts

```diff
@@ -154,19 +172,14 @@ tasks.whenTaskAdded {
     }
 }
 
-// Disabling default jar task as it is overridden by shadowJar
-tasks.named("jar").configure {
-    enabled = false
-}
-
 val ensureDependenciesAreInlined by tasks.registering {
```

Comment:
Is this task even doing what we were thinking it was doing? Is it doing it adequately?

You said that our pom and gradle metadata dependencies were being exposed as direct dependencies, right?

Reply:
I played around with it a bit, I there was nothing I could do to actually make this thing fail. I think the shadow plugin has improved to where it is harder to mess this stuff up now. So I have removed this task